pkg/elements/document.go

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright 2025 The Protobom Authors
+
+package elements
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/protobom/protobom/pkg/sbom"
+)
+
+var DocumentType = cel.ObjectType("protobom.protobom.Document")
+
+type Document struct {
+	*sbom.Document
+}
+
+// ConvertToNative implements ref.Val.ConvertToNative.
+func (d Document) ConvertToNative(typeDesc reflect.Type) (interface{}, error) {
```

Comment:
Are you ok with using `any`?

```suggestion
func (d Document) ConvertToNative(typeDesc reflect.Type) (any, error) {
```

Reply:
No problem, I've changed them all :+1: